Dockerfile

```diff
@@ -0,0 +1,93 @@
+                   file                    \
+                   gcc-c++                 \
+                   git                     \
+                   libffi-devel            \
+                   libtool                 \
+                   libxml2-devel           \
+                   libxslt-devel           \
+                   libyaml-devel           \
+                   make                    \
+                   memcached               \
+                   net-tools               \
+                   nodejs                  \
+                   openssl-devel           \
+                   patch                   \
+                   rh-postgresql94-postgresql-server \
+                   rh-postgresql94-postgresql-devel  \
+                   readline-devel          \
+                   sqlite-devel            \
+                   which                   \
+                   &&                      \
+    yum clean all
+
+# Add persistent data volume for postgres
+VOLUME [ "/var/opt/rh/rh-postgresql94/lib/pgsql/data" ]
+
+# Download chruby and chruby-install, install, setup environment, clean all
+RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz && \
+    cd chruby-0.3.9 && make install && scripts/setup.sh && \
+    echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc && \
+    echo "source /usr/local/share/chruby/chruby.sh" >> ~/.bashrc && \ 
+    curl -sL https://github.com/postmodern/ruby-install/archive/v0.6.0.tar.gz | tar xz && \
+    cd ruby-install-0.6.0 && make install && ruby-install ruby 2.2.4 -- --disable-install-doc && \
+    echo "chruby ruby-2.2.4" >> ~/.bash_profile && \
+    rm -rf ~/ruby* && rm -rf /usr/local/src/* && yum clean all
+
+## Environment for scripts
+RUN echo "export BASEDIR=/manageiq" > /etc/default/evm && \
+    echo "export PATH=$PATH:/opt/rubies/ruby-2.2.4/bin:/opt/rh/rh-postgresql94/root/bin:/opt/rh/rh-postgresql94/root/usr/libexec" >> /etc/default/evm && \
+    echo "[[ -s /etc/default/evm_postgres ]] && source /etc/default/evm_postgres" >> /etc/default/evm && \
+    echo "export APPLIANCE_PG_SCL_NAME=rh-postgresql94" > /etc/default/evm_postgres && \
+    echo "export APPLIANCE_PG_SERVICE=rh-postgresql94-postgresql" >> /etc/default/evm_postgres && \
+    echo "export APPLIANCE_PG_DATA=/var/opt/rh/rh-postgresql94/lib/pgsql/data" >> /etc/default/evm_postgres && \
+    echo "export APPLIANCE_PG_CHECKDB=/opt/rh/rh-postgresql94/root/usr/libexec/postgresql-check-db-dir" >> /etc/default/evm_postgres && \
+    echo "[[ -s /opt/rh/\${APPLIANCE_PG_SCL_NAME}/enable ]] && source /opt/rh/\${APPLIANCE_PG_SCL_NAME}/enable" >> /etc/default/evm_postgres
+
+## Create basedir, GIT clone miq (shallow)
+#RUN mkdir -p /manageiq && git clone --depth 1 https://github.com/ManageIQ/manageiq /manageiq
+RUN mkdir -p /manageiq
+ADD . /manageiq
+
+## Change WORKDIR to clone dir, copy docker_setup, start all, docker_setup, shutdown all, clean all
+WORKDIR /manageiq
```

Comment:
@bdunne IIUC that path is used due to the use of httpd in the current appliance, 
SO I think the discussion of using httpd is relevant more for the container appliance use case , currently this work is mostly related to developer use case.
This point is valid and will be discussed as a part of the appliance approach.